content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -613,7 +613,7 @@ version as kubeadm or one version older.
 
```

Comment:
^ change this :

> Similarly to the Kubernetes version, kubeadm can be used with a kubelet version that is the same
version as kubeadm or one version older.

to:

> Similarly, to the Kubernetes version, kubeadm can be used with a kubelet version that is the same
version as kubeadm or up to 3 versions older.